odoo-shippable/scripts/build-image.sh

```diff
@@ -20,6 +20,8 @@ PYTHON_PPA_REPO="deb http://ppa.launchpad.net/fkrull/deadsnakes/ubuntu trusty ma
 PYTHON_PPA_KEY="http://keyserver.ubuntu.com:11371/pks/lookup?op=get&search=0x5BB92C09DB82666C"
 VIM_PPA_REPO="deb http://ppa.launchpad.net/pkg-vim/vim-daily/ubuntu trusty main"
 VIM_PPA_KEY="http://keyserver.ubuntu.com:11371/pks/lookup?op=get&search=0xA7266A2DD31525A0"
+PY36_PPA_REPO="deb http://ppa.launchpad.net/jonathonf/python-3.6/ubuntu trusty main"
```

Comment:
I'm not sure but I think that the current PPA REPO has python3 and is not required add other one